### DISCUSSION TITLE
inbuilt microphone

### DISCUSSION THREAD

**Discussion by thorbian** [SpiralLinux/SpiralLinux-project#492] [Q&A] [answered]
I have terrible sound on my inbuilt microphone. I have a windows partition on the same computer where the sound is fine. So I searched for solutions and found a solution, which recommended installing pipewires with backports.
I looked in to and got this output,
```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Selected version '1.2.4-1~bpo12+1' (Debian Backports:stable-backports [amd64]) for 'pipewire'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 pipewire : Depends: libpipewire-0.3-modules (= 1.2.4-1~bpo12+1) but 0.3.65-3+deb12u1 is to be installed
            Depends: pipewire-bin (= 1.2.4-1~bpo12+1)
E: Unable to correct problems, you have held broken packages.

```
Is this something with the special configuration of spiral or something completely different?

**Comment by geckolinux** [comment, marked as answer]
> Is this something with the special configuration of spiral or something completely different?

Hi there, no, that's not specific to SpiralLinux since it only uses Debian repositories. I also installed Pipewire from Backports and had similar issues, but I was eventually able to make it work. You could probably look into the topic of APT pinning and/or ask for help on a Debian forum or Backports mailing list.
